react/ActionMenu/index.jsx

```diff
@@ -101,6 +102,29 @@ const ActionMenu = ({
   anchorElRef,
   containerElRef
 }) => {
+  const webviewIntent = useWebviewIntent()
+
+  useEffect(() => {
+    webviewIntent &&
+      webviewIntent.call('setFlagshipUI', {
+        bottomBackground: getCssVariableValue('white'),
+        bottomColor: 'dark',
+        topOverlay: getCssVariableValue('overlay'),
+        topBackground: getCssVariableValue('paperBackgroundColor'),
+        topColor: 'light'
+      })
+
+    return () =>
+      webviewIntent &&
+      webviewIntent.call('setFlagshipUI', {
+        bottomBackground: getCssVariableValue('paleGrey'),
+        bottomColor: 'dark',
+        topOverlay: 'transparent',
+        topBackground: getCssVariableValue('white'),
+        topColor: 'dark'
+      })
+  }, [webviewIntent])
```

Comment:
adding feature without unit tests make code more fragile.

Let's test this.

Do you want some help, achieving this?

Reply:
@trollepierre I think it's hard to test this as it's purely declarative. Maybe I can test the call is made once on mount and once on unmount, but not verify what values it calls. Or else the test will basically test the implementation and will need changed when the implementation changes

Reply:
I can test the call is made once on mount and once on unmount, but not verify what values it calls. => 🆗

Reply:
@trollepierre we talked about it with @JF-Cozy and found a good way to test it, we're going to create a single hook that abstract this one. It will be used everywhere and will have a strong test